src/cli.js

```diff
@@ -46,8 +46,10 @@ const runCli = (options) => {
         if (duplicatedPropertyKeys != null) {
           if (duplicatedPropertyKeys.length === 0) {
             console.log(chalk.green(`No duplicated property keys found in ${options.src}.`))
+            return 0;
           } else {
-            console.log(chalk.red(`The following duplicated property keys have been detected in ${options.src}:\n${duplicatedPropertyKeys.map(duplicatedPropertyKey => `${duplicatedPropertyKey} (${duplicatedPropertyKey.occurrence})`).join('\n')}`))
+            console.error(chalk.red(`The following duplicated property keys have been detected in ${options.src}:\n${duplicatedPropertyKeys.map(duplicatedPropertyKey => `${duplicatedPropertyKey} (${duplicatedPropertyKey.occurrence})`).join('\n')}`))
+            return 1;

```

Comment:
The return code should not be 1, as this status will also be returned if an exception is thrown. I would suggest 101, such that validation errors can clearly be distinguished from program errors.

Please also remove the semicolon, as these are not desired according to https://standardjs.com/ code conventions, which I am following here.